common/src/main/java/com/wynntils/services/chat/ChatTabService.java

```diff
@@ -54,6 +54,9 @@ public List<ChatTab> getChatTabs() {
     }
 
     public ChatTab getTab(int index) {
+        if (index < 0 || index >= getChatTabs().size()) {
+            return null;
+        }
```

Comment:
```suggestion
        if (index < 0 || index >= getChatTabs().size()) return null;
```

And they can be omitted from the `setFocusedTab` change too